### PR TITLE
Add Threshold Network Token (T) to Base

### PR DESCRIPTION
**Pull request by SYLMingYue**
Adding Threshold Network Token (T) on Base List

Basescan: https://basescan.org/address/0x26f3901ac8a79c50fb0d8289c74f0d09adc42e29

@roberto-bayardo

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Can you regenerate the L2 token using the factory in our current docs / instructions? (https://docs.base.org/base-contracts/#l2-contract-addresses) 

Unfortunately the predeploy generates token addresses that conflict with other tokens on Optimism.

**Comment by SYLMingYue** [comment]
> Can you regenerate the L2 token using the factory in our current docs / instructions? (https://docs.base.org/base-contracts/#l2-contract-addresses)
> 
> Unfortunately the predeploy generates token addresses that conflict with other tokens on Optimism.

All right, Is there any way to withdraw my bridged tokens "T" from Base?

**Comment by roberto-bayardo** [comment]
Yes the tokens will always remain permissionlessly bridgeable back to the source token on the L1 via the standard bridge contract.

**Comment by SYLMingYue** [comment]
> Yes the tokens will always remain permissionlessly bridgeable back to the source token on the L1 via the standard bridge contract.

Very good! Is there a tutorial or documentation for this type of bridging? I can't find relevant information on the base official website

**Comment by roberto-bayardo** [comment]
https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge-erc20

**Comment by SYLMingYue** [comment]
> https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge-erc20



> https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge-erc20

thank you very much! I'll try to do some research on this document first, and if I have any questions, I'll come back to ask you here, okay?

File: data/T/data.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     },
     "goerli": {
       "address": "0x3f16380656cAE45D3f80D8833682d2b606eD094A"
+    },
+    "base": {
+      "address": "0x26f3901ac8a79c50fb0d8289c74f0d09adc42e29"
```
Comment (roninjin10) [inline data/T/data.json:19]: needs base approval

**Review by roninjin10 — APPROVED** [review]
lgtm needs base approval

**Comment by hamdiallam** [comment]
@SYLMingYue Have you had a chance to regenerate this token address yet?

**Comment by SYLMingYue** [comment]
@hamdiallam Awfully sorry! Actually i'm not the creater of the T token, So i think it will be better to let the token admin do this work.

**Pull request closed without merging**